ExampleMod/ExampleMod.Networking.cs

```diff
@@ -14,6 +15,8 @@ namespace ExampleMod
 	// This class extends from the Mod class as seen in ExampleMod.cs. Make sure to extend from the mod class, ": Mod", in your own code if using this file as a template for you mods Mod class.
 	partial class ExampleMod
 	{
+		public static LocalizedText LoggerText { get; private set; }
```

Comment:
We don't use localizations in logs as they're not player facing, and can make it harder for the developers to diagnose issues